lib/DDG/Spice/Landsat.pm

```diff
@@ -0,0 +1,32 @@
+package DDG::Spice::Landsat;
+# ABSTRACT: Gets satellite imagery for given lat/lon
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries 'landsat 40.7033127,-73.979681';
+description 'NASA Landsat imagery';
+name 'Landsat';
+attribution web => ['http://dylansserver.com','Dylan Lloyd'],
+            email => ['dylan@dylansserver.com','Dylan Lloyd'],
+            github => ['https://www.github.com/nospampleasemam', 'Dylan Lloyd'];
+
+spice to => 'https://api.data.gov/nasa/planetary/earth/imagery?api_key={{ENV{DDG_SPICE_NASA_APIKEY}}}&lat=$1&lon=$2&dim=0.5';
+
+spice wrap_jsonp_callback => 1;
+
+triggers query_lc => qr/^
```

Comment:
Can we switch this to use trigger words and a regex guard? Would be a little more efficient!